plugin/src/test/java/org/opensearch/ml/rest/RestMLRAGSearchProcessorIT.java

```diff
@@ -716,8 +716,7 @@ public void testBM25WithOpenAIWithImage() throws Exception {
         requestParameters.timeout = 60;
         requestParameters.imageFormat = "jpeg";
         requestParameters.imageType = "url";
-        requestParameters.imageData =
-            "https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Gfp-wisconsin-madison-the-nature-boardwalk.jpg/2560px-Gfp-wisconsin-madison-the-nature-boardwalk.jpg"; // imageContent;
+        requestParameters.imageData = "https://images.unsplash.com/photo-1506905925346-21bda4d32df4?w=800";
```

Comment:
Just curious why do need to download the image? What is the goal of this integration test?
Can we keep an image in our resources folder? What if this image link breaks again?